src/frontend/komponenter/Felleskomponenter/RegistrerDødsfallDato/RegistrerDødsfallDatoModal.tsx

```diff
@@ -44,6 +44,7 @@ const RegistrerDødsfallDatoModal = ({ lukkModal, person, erLesevisning }: IProp
                         label={'Dødsdato'}
                         visFeilmeldinger={skjema.visFeilmeldinger}
                         readOnly={erLesevisning}
+                        kanKunVelgeFortid
```

Comment:
Før kunne man velge dato i fremtiden, men det gir ikke mening for dødsfall, så jeg har lagt på at man kun kan velge dato i fortiden
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/d277f6a9-9021-4b00-a77a-33a07b86f767)